.github/workflows/ci_python.yml

```diff
@@ -105,15 +105,24 @@ jobs:
             dnf -y install clang protobuf-devel lld
             clang --version
             protoc --version
+      - name: Determine Version
+        if: github.event_name != 'release'
+        run: |
+          pip install tomlkit
+          VERSION=$(python ../scripts/get_version.py pyproject.toml project.version)
+          echo "WHEEL=dist/kaskada-${VERSION}-cp38-abi3-manylinux_2_28_x86_64.whl"
```

Comment:
In CI, extract the `project.version` from the `pyproject.toml` and use that to generate the path to the `WHEEL`.